org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/relaxng/xml/validator/RelaxNGErrorCode.java

```diff
@@ -92,6 +119,8 @@ public static Range toLSPRange(XMLLocator location, RelaxNGErrorCode rngCode, Ob
 		int offset = location.getCharacterOffset() - 1;
 		// adjust positions
 		switch (rngCode) {
+
+			// XML Validation based on RNG, RNC errors

```

Comment:
I think we can add `unexpected_element_required_elements_missing` and `incomplete_element_required_elements_missing` here to have range as the start tag name

Reply:
Have you samples which causes those errors?

Reply:
Ohh sorry I just realized that this is unrelated to this PR, it is just the plural version of `unexpected_element_required_element_missing`.

Reply:
I think this is good to merge!

Reply:
> Ohh sorry I just realized that this is unrelated to this PR, it is just the plural version of unexpected_element_required_element_missing.

Indeed the RNG validator uses another error code for that.